build_tools/docker/context/install_iree_deps.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -euo pipefail
+
+LLVM_VERSION="$1"
+
+declare -a PACKAGES=(
+    "clang-${LLVM_VERSION}"
+    "lld-${LLVM_VERSION}"
+    # IREE transitive dependencies
+    libsdl2-dev
```

Comment:
We might be able to remove sdl, I think the only thing using it was https://github.com/iree-org/iree-samples/tree/main/cpp/vulkan_gui

Reply:
Rebuilding, here I come :cry: The samples docker container doesn't install it though. Should we move it there? Maybe I should test the samples build on this PR?

Reply:
The iree-samples repository C/C++ code is untested. We moved the vulkan gui sample from iree-org/iree to iree-org/iree-samples _because_ it involved complicated dependencies like SDL2. I'd have to trace through the history in these Docker files to see where/when libsdl2-dev was added... maybe it helped building with `IREE_BUILD_SAMPLES=ON` at some point...

Reply:
https://github.com/iree-org/iree/pull/3065 ah yeah, added here - very old. I think we can remove `libsdl2-dev` now.

Reply:
Ok, I'm going to remove it then. If something breaks postsubmit and no presubmit, that's on you though :-P

Reply:
Ugh apparently sdl2 was implicitly giving us `vulkaninfo`. And the apt package changed names between Ubuntu 18.04 and 22.04.... I'm just going to revert and leave a note that it could probably be removed next time someone's rebuilding. My workstation has run out of disk a couple times building these and I need to clean it up, so I wouldn't even get cache hits if I rebuilt yet again.

Reply:
Aw, thanks for checking at least!